Gui/GeometryGUI/Area/AreaFormulas.java

```diff
@@ -0,0 +1,25 @@
+package GeometryGUI.Area;
+
+
+/**
+ * This class contains methods that return the area's of different shapes. 
+ * @author scohen40
+ */
+public class AreaFormulas {
```

Comment:
Why is this class duplicated?

Reply:
That was the other thing I forgot that I didn't get a chance to ask you, last class. The file can't see it in the other package for some reason.

Reply:
The Gui package, for some reason, can't see it in the other package.